binaries/scootapi/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"git.apache.org/thrift.git/lib/go/thrift"
+	"github.com/scootdev/scoot/scootapi/client"
+	"log"
+)
+
+func main() {
+	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
```

Comment:
Yup, pants doesn't care about it.

So, I think we need to talk more at some point about the difference between the local scoot interface (that pants needs) and the cloud scoot api that Iron will use to run things.

This code is for the latter.